src/clients/ntp-client_proposal.rb

```diff
@@ -436,8 +458,11 @@ def ui_try_save
 
       rv = Write(argmap)
 
-      server = Convert.to_string(UI.QueryWidget(Id(:ntp_address), :Value))
-
+      server = if select_ntp_server
```

Comment:
will it work below? especially https://github.com/yast/yast-ntp-client/pull/156/files#diff-637cd929308624ec8989c144f1d7b7c6R484 where it will write this first server? Also I am not sure if it is proper behavior if you have set of servers that it query only the first one.

Reply:
I have removed it.